### DISCUSSION TITLE
我们是否应该上传至 Google Fonts 或 Fontsource？

### DISCUSSION THREAD

**Discussion by mogeko** [atelier-anchor/smiley-sans#60] [Polls]
我们是否应该将得意黑发布到 [Google Fonts](https://fonts.google.com) 或 [Fontsource](https://fontsource.org)？

我是一名前端开发者，我希望在我的网站上使用得意黑。目前，我唯一使用得意黑的方式是下载字体包，然后将它作为静态资产一同提交到网站的 GitHub 仓库中。并且我还需要定期手动为其升级。但如果我们将得意黑上传到了 Google Fonts，我们前端仅需要一句 CSS 就可以使用得意黑了。

更进一步，得意黑作为艺术字体，我们可能仅需要使用其中的几个字形，对于这种情况，引用整个字体包就非常的不划算；为此，我们可以利用 [Google Fonts 的 Subsetting](https://fonts.google.com/knowledge/glossary/subsetting) #4 来动态的选择我们需要封包的字。

另一方面，Google Fonts 对字体的打包格式有一定对要求，发布到 Google Fonts 或 Fontsource 无疑会增加开发者们的工作量。

因此，我想发起一次投票来听听大家的意见。同时，如果开发者允许的话；我会着手研究发布的流程；我的最终目标是希望通过 GitHub Actions 完成这所有的工作。

一些参考链接：

- [Google Fonts Guide](https://googlefonts.github.io/gf-guide/)
- [Generic Font Packager (Fontsource)](https://github.com/fontsource/fontsource/blob/main/scripts/generic/README.md)

**Comment by howlingFounts** [comment]
我还真给 Google Fonts 提了个 Issue: https://github.com/google/fonts/issues/5550

**Comment by itsdapi** [comment]
非常需要！！！

**Comment by Alfxjx** [comment]
现在本地加载还是不够优雅 要是可以走cdn就好了

**Comment by cugxuan** [comment]
需要！！！

**Comment by XieyilinMain** [comment]
Google Fonts网站的MD3设计本来就舒服，在这个上面获取字体的体验不是一般的好
